### PR TITLE
[TAPAS] Tiny fix

### PR DESCRIPTION
**Pull request by NielsRogge**
# What does this PR do?

This PR includes a finy fix for TAPAS. Namely, when `config.select_one_column` is set to `False`, the model should not recompute the token logits. 

Relevant to #13393

### CODE REVIEW

**Comment by LysandreJik** [comment]
Can you include a test to ensure the behavior doesn't happen in the future?
